addons/web_tour/static/src/js/tip.js

```diff
@@ -211,15 +212,17 @@ var Tip = Widget.extend({
             // important to only remove/add the class when necessary to not
             // notify a DOM mutation which could retrigger this function.
             const $oldLocation = this.$el.parent();
-            if (!$location.is($oldLocation)) {
-                $oldLocation.removeClass('o_tooltip_parent');
-                const cssPosition = $location.css("position");
-                if (cssPosition === "static" || cssPosition === "relative") {
-                    $location.addClass("o_tooltip_parent");
+            if(!this.tip_opened) {
```

Comment:
It annoys me to delay the merge of your work for such a small issue but here there is a missing space. You really have to install a linter to avoid those kind of issues :) I already told you in my first review:

> Can you install a linter and finx the errors you introduce? https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines

This is also still true:

> Also this needs to target the version where the bug was introduced please, thanks.

Reply:
@qsm-odoo i have changed the base of the branch to saas-13.4 as we can reproduce this issue in saas-13.4